src/components/Header/components/QuickSearch/QuickSearch.module.scss

```diff
@@ -19,6 +21,7 @@
 
   & :global(.MuiInputBase-root) {
     padding: 2px 10px;
+    color: var(--mui-palette-primary-contrastText);
```

Comment:
Unfortunately, this breaks the text color of QuickSearch in the RecIM Participant Search:
![image](https://github.com/gordon-cs/gordon-360-ui/assets/35319956/8b7f392b-5971-400f-9874-4ad2b5fd2b4b)

We might have to come up with a different way to style the QuickSearch so that it's visible on either background color.

Reply:
Thank you for your review! I'm wondering how to get the text color of the parent. The parent's text color usually contrasts with its background color, I think it would be better to use the parent's text color instead of using primary-contrastText color.

Reply:
I just found that color: inherit; get the parent's color.